### ISSUE TITLE
[match-description] Reports on message ending with emoji

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [gajus/eslint-plugin-jsdoc#1433]
## Expected behavior

The following code should not report any errors by default:

```js
/**
 * Example text. 🙂
 */
export const example = () => { };
```

## Actual behavior

```plaintext
/Users/josh/repos/repros/index.js
  2:1  error  JSDoc description does not satisfy the regex pattern  jsdoc/match-description
```

## ESLint Config

```js
import jsdoc from "eslint-plugin-jsdoc";

export default [
    {
        plugins: { jsdoc },
        rules: {
            "jsdoc/match-description": "error",
        },
    },
];
```

## ESLint sample

See earlier.

## Environment

- Node version: 24.3.0
- ESLint version: 9.31.0
- `eslint-plugin-jsdoc` version: 51.4.1

**Comment by brettz9** [comment]
Released as a breaking change as called for using the `v` regex flag.

**Comment by JoshuaKGoldberg** [comment]
⚡ thanks for the quick fix!
